spec/paper_trail/request_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module PaperTrail
+  ::RSpec.describe(Request, versioning: true) do
```

Comment:
If you have the time, I'd probably prefer to have the other `@api public` methods tested as well here, such as `controller_info`, `controller_info=`, `enabled_for_controller?`, `enabled_for_controller=`, and `whodunnit=`.